.github/workflows/sonarcloud.yaml

```diff
@@ -0,0 +1,53 @@
+name: Sonarcloud Scan
+
+on:
+  push:
+    branches:
+      - 'main'
```

Comment:
I think the branch should be master instead of main.

Reply:
done